eva-peng/week4/wed/httparty/views/layout.erb

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Movie</title>
+  <link rel="stylesheet" href="/css/style.css" />
+</head>
+<body>
+  
+  <header>
+    <nav>
+      <h1>HTTParty</h1>
+      <hr>
```

Comment:
make sure you add a link to the lookup. 

When i go to your home page, i cannot navigate to the lookup page unless i write `/lookup` into the address bar.

Reply:
you also want to add a link to go back to the `lookup` page